std-bits/table/src/main/java/org/enso/table/data/column/operation/unary/IsInfiniteOperation.java

```diff
@@ -0,0 +1,45 @@
+package org.enso.table.data.column.operation.unary;
+
+import org.enso.table.data.column.builder.BoolBuilder;
+import org.enso.table.data.column.builder.Builder;
+import org.enso.table.data.column.operation.UnaryOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemAggregator;
+import org.enso.table.data.column.storage.ColumnDoubleStorage;
+import org.enso.table.data.column.storage.ColumnLongStorage;
+import org.enso.table.data.column.storage.ColumnStorage;
+
+public class IsInfiniteOperation extends AbstractUnaryBooleanOperation {
+  public static final UnaryOperation INSTANCE = new IsInfiniteOperation();
+
+  private IsInfiniteOperation() {
+    super(UnaryOperation.IS_NAN, true);
+  }
+
+  @Override
+  public boolean canApply(ColumnStorage storage) {
+    return storage.getType().isNumeric();
+  }
+
+  @Override
+  protected void applyLong(ColumnLongStorage longStorage, Builder builder, MapOperationProblemAggregator problemAggregator) {
+    var boolBuilder = (BoolBuilder)builder;
+    UnaryOperation.applyOverLongStorage(longStorage, true, builder, (isNothing, value) -> boolBuilder.appendBoolean(false));
+  }
```

Comment:
The old implementation for this case used to be O(1).

We were inheriting the `isMissing` `BitSet` and allocating an all-false empty `BitSet` for the values, both O(1) operations.

Now the mapping is O(N).

I appreciate that it is a bit cleaner now, but can we try to still retain such special cases to ensure efficient computation?

Reply:
Missed that - will make this O(1) for Long.